.github/workflows/ui.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: '1.21.0'
```

Comment:
@n8maninger how about here on renterd, stable like hostd? renterd was working fine, I was just trying to make things consistent.